shared/desktop/app/installer.desktop.js

```diff
@@ -1,59 +1,92 @@
 // @flow
 import * as SafeElectron from '../../util/safe-electron.desktop'
+import fs from 'fs'
+import path from 'path'
 import exec from './exec.desktop'
 import {keybaseBinPath} from './paths.desktop'
 import {quit} from './ctl.desktop'
-import {isLinux, isWindows} from '../../constants/platform'
+import {isDarwin} from '../../constants/platform'
 import logger from '../../logger'
-import UserData from './user-data.desktop'
 
-import type {InstallResult} from '../../constants/types/rpc-gen'
+const file = path.join(SafeElectron.getApp().getPath('userData'), 'installer.json')
```

Comment:
in dev mode this path is ~/Library/Application Support/Electron (and not Keybase) due to some other issue. not worth fixing now but if you want to test this the installer.json file isn't in Keybase